about_binaries.exs

```diff
@@ -31,4 +31,66 @@ defmodule AboutBinaries do
     contatenated = "Hello " <> "world"
     assert contatenated == __?
   end
+
+  @tag :focus
+  think "Numbers can be represented as bits" do
+    bit_representation = fn (integer) -> Integer.to_string(integer, 2) end
+    assert bit_representation.(1) == "1"
+    assert bit_representation.(2) == "10"
+    assert bit_representation.(3) == "11"
+    assert bit_representation.(4) == "100"
+  end
+
+  @tag :focus
+  think "255 is the max value we can represent with 8 bits (1 byte)" do
+    bit_representation = fn (integer) -> Integer.to_string(integer, 2) end
```

Comment:
Since this is accessed by multiple examples, you could extract a named function in this module.